lib/awoo.js

```diff
@@ -5,8 +5,14 @@ const write = require('./write')
 const { generateConfig } = require('./config')
 
 async function siteUse (fn, opts = {}) {
-  const plugin = await fn(opts)
-  this.hooks.use(plugin)
+  try {
+    const plugin = await fn(opts)
+    this.hooks.use(plugin)
+  } catch(err) {
+    debug(`plugin ${fn.name ||fn} init failed :`)
+    debug(err);
+    process.exit(1); // don't continue
```

Comment:
Would be cool to also just throw the error publicly (i.e. not only in debug) if we just hard exit here.

Reply:
yep... [debug()](https://github.com/visionmedia/debug) really doesn't have a notion of _different levels_ of debugging.. unlike [winston](https://github.com/winstonjs/winston) for instance : It's really only a switch on/off
So, what should we use instead ? console.err() ?

Reply:
Yeah, that'll work.